Remotion/Web/Development.WebTesting/WebDriver/BrowserConfigurationExtensions.cs

```diff
@@ -37,7 +38,8 @@ public static bool IsChrome ([NotNull] this IBrowserConfiguration browserConfigu
     {
       ArgumentNullException.ThrowIfNull(browserConfiguration);
 
-      return browserConfiguration is IChromeConfiguration;
+      return browserConfiguration is IChromeConfiguration
```

Comment:
why us `is IsChromeConfiguration` here while changing this very code to `IscChrome()` in line 65?